src/ui/Card/Card.tsx

```diff
@@ -0,0 +1,79 @@
+import { cva, VariantProps } from 'cva'
+import React from 'react'
+
+const card = cva(['border border-ds-gray-secondary'])
+interface CardProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    VariantProps<typeof card> {}
+
+const Card = React.forwardRef<HTMLDivElement, CardProps>(
```

Comment:
Swap this to `CardRoot` from `Card`

```suggestion
const CardRoot = React.forwardRef<HTMLDivElement, CardProps>(
```